source/unified-test-format/schema-1.3.json

```diff
@@ -263,13 +385,17 @@
         "name": { "type": "string" },
         "object": { "type": "string" },
         "arguments": { "type": "object" },
+        "ignoreResultAndError": { "type": "boolean" },
         "expectError":  { "$ref": "#/definitions/expectedError" },
         "expectResult": {},
         "saveResultAsEntity": { "type": "string" }
       },
       "allOf": [
         { "not": { "required": ["expectError", "expectResult"] } },
-        { "not": { "required": ["expectError", "saveResultAsEntity"] } }
+        { "not": { "required": ["expectError", "saveResultAsEntity"] } },
+        { "not": { "required": ["ignoreResultAndError", "expectResult"] } },
+        { "not": { "required": ["ignoreResultAndError", "expectError"] } },
+        { "not": { "required": ["ignoreResultAndError", "saveResultAsEntity"] } }
```

Comment:
New `invalid` tests:

 * `operation-ignoreResultAndError-conflicts_with_expectResult.yml`
 * `operation-ignoreResultAndError-conflicts_with_expectError.yml`
 * `operation-ignoreResultAndError-conflicts_with_saveResultAsEntity.yml`

Reply:
I realize you added these in `ignoreResultAndError.yml` but they should be split up into separate files. We currently cannot assert multiple JSON schema failures in a single file.